src/test/RwaUrn.t.sol

```diff
@@ -308,6 +308,34 @@ contract RwaExampleTest is DSTest, DSMath, TryPusher {
         assertEq(dai.balanceOf(address(inConduit)), 0 ether);
     }
 
+    function test_partial_repay_fuzz(uint256 drawAmount, uint256 wipeAmount, uint256 drawTime, uint256 wipeTime) public {
+        // Convert to reasonable numbers
+        drawAmount = (drawAmount % 300 ether) + 100 ether;      // 100-400 ether
+        wipeAmount = wipeAmount % drawAmount;                   // 0-drawAmount ether
+        drawTime = drawTime % 15 days;                       // 0-15 days
+        wipeTime = wipeTime % 15 days;                       // 0-15 days
+
+        usr.lock(1 ether);
+
+        hevm.warp(now + drawTime);
+        jug.drip("acme");
```

Comment:
I just merged https://github.com/livnev/rwa-example/pull/26, if you rebase you could remove this line.

Reply:
Actually the `drip` is still useful if we want to recalculate the new `rate` before calling `draw` or `wipe`. Otherwise we need to replicate the `rpow` function and do all the math.
The thing I'm wondering if this test should be more focused right to the edges. So basically making a fuzzing that always tries to repay the whole debt (including the fees generated), that's probably the most interesting case to test due possible rounding errors. Like proving that for a different set of values we are always able to leave the vault empty.